packages/core/src/Layout/Layout.ts

```diff
@@ -41,18 +63,24 @@ class Layout {
       };
       pageSources = new PageSources();
       this.layoutPageBodyVariable = `{{${uuidv4()}-${uuidv4()}}}`;
-      const layoutVariables = { [LAYOUT_PAGE_BODY_VARIABLE]: this.layoutPageBodyVariable };
+      const layoutVariables = {
+        [LAYOUT_PAGE_BODY_VARIABLE]: this.layoutPageBodyVariable,
+      };
 
-      const { variableProcessor, pluginManager, siteLinkManager } = this.config;
+      const { variableProcessor, pluginManager, siteLinkManager }: {
+        variableProcessor: VariableProcessor, pluginManager: PluginManager, siteLinkManager: SiteLinkManager
```

Comment:
Is this line necessary? These types might be implicit since we are destructuring from `this.config` which defines the attributes as having these types. Correct me if I'm wrong 😅